test/e2e/suite/conformance/certificates/tests.go

```diff
@@ -881,6 +883,11 @@ func (s *Suite) Define() {
 
 			By("Sanity-check the issued Certificate")
 			err = f.Helper().ValidateCertificate(testCertificate, validations...)
+			if err != nil {
+				pretty.Logln("cert:", testCertificate)
+				testCertificate, _ = f.Helper().WaitForCertificateReadyAndDoneIssuing(testCertificate, time.Minute*5)
+				pretty.Logln("re-fetched:", testCertificate)
```

Comment:
@jakexks @wallrj these print statements should give some more insights into what causes the "Expected certificate expiry date to be 2026-08-05 17:22:43 +0100 BST, but got 2026-08-05 16:23:13 +0000 UTC" error.
I'm however not able to replicated the issue on my machine. Could someone try this and share the results?